emrun.py

```diff
@@ -454,7 +454,7 @@ def serve_forever(self, timeout=0.5):
     self.print_all_messages()
 
   def handle_error(self, request, client_address):
-    err = sys.exc_info()[1][0]
+    err = sys.exc_info()[1].args[0]
```

Comment:
Hmm, do you know more about this? I wonder if this has changed in some python version, or if this should not have worked at all? I don't think I saw an exception being raised with the old code on accessing a missing field.

Reply:
This is an odd Python 2-specific behavior.

```py
Exception('message')[0] # only on Python 2
Exception('message').args[0] # Python 2/3
```

Reply:
Oh, interesting indeed.. thanks!